hashes/zkevm-keccak/src/keccak_packed_multi.rs

```diff
@@ -1954,6 +1950,350 @@ pub fn keccak_phase0<F: Field>(
     }
 }
 
+/// Witness generation in `FirstPhase` for a keccak hash digest without
+/// computing RLCs, which are deferred to `SecondPhase`.
+pub fn keccak_phase0_with_flags<F: Field>(
```

Comment:
I wonder if this could be combined with the original fn?  I had to locally check the actual diff to see what had changed, but it's a pretty small proportion of such a large function.

Reply:
Combined